packages/selfcare-client-users-updater-consumer/README.md

```diff
@@ -0,0 +1,15 @@
+# Selfcare client users updater
+
+This service is responsible for managing updates to client user values.
+
+## Main Features
+
+- Removing the admin role from a client for a user who is no longer an admin in a specific tenant.
+- Managing users who are no longer part of a tenant.
```

Comment:
`Managing` sounds more generic 🤔 which is the action if users is was removed from tenant?